python/test/t_FFTW_ND_std.py

```
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+from openturns import *
```

Ruff (0.15.20)

[error] `from openturns import *` used; unable to detect undefined names

(F403)